prime-router/metadata/organizations-local.yml

```diff
@@ -128,15 +128,47 @@
   - name: pa-phd
     description: Pennsylvania Department of Health
     services:
-      - name: elr-redox-debug
+      - name: elr-bucks-local
+        topic: covid-19
```

Comment:
We need to use different Redox destinations for each PA county. It's sad, but true.

Reply:
We are approaching the time when we need to move everything in organizations.yml into the database, or, at least outside of the build, so we can modify without a deploy.